registry/cloudflare-workers/index.js

```diff
@@ -26,44 +37,34 @@ const doWorkerDeploy = async ({ zone_id, script_path }) => {
     result
   }
 }
-/*
-const removeWorker =  ({zone_id}) => {
-	
-	const AUTH_KEY = process.env.CLOUDFLARE_AUTH_KEY
-	const EMAIL = process.env.CLOUDFLARE_EMAIL
-	const url = `https://api.cloudflare.com/client/v4/zones/${zone_id}/workers/script`
 
-	const headers = {
-		"X-Auth-Email": `${EMAIL}`,
-		"X-Auth-Key": `${AUTH_KEY}`,
-		"Content-Type": "application/json"
-	}
+const doWorkerDeploy = async ({ zone_id, script_path }) => {
+  const AUTH_KEY = process.env.CLOUDFLARE_AUTH_KEY
```

Comment:
@pinkerton add some documentation so users know they must have their auth key and email set as an environment variable

`export CLOUDFLARE_EMAIL=${email}`
`export CLOUDFLARE_AUTH_KEY=${auth_key}`